README.md

```diff
@@ -1,4 +1,22 @@
-# Sui README
+<p align="center">
+  <a href="https://docs.sui.io/">
+    <img src="https://github.com/MystenLabs/sui/blob/main/doc/static/Sui_Icon_Brand.png" alt="Logo" width="80" height="80">
+  </a>
+  
+  <h3 align="center">Sui</h3>
+
+  <p align="center">
+    Welcome to Sui, a next-generation smart contract platform with high throughput, low latency, and an asset-oriented programming model.
+    <br />
+    <a href="https://docs.sui.io/"><strong>Explore more about Sui »</strong></a>
+    <br />
+    <br />
+    <a href="https://discord.gg/sui">Community</a>
+    ·
+    <a href="https://docs.sui.io/">Docs</a>
+  </p>
+</p>
+<hr>
 
 |             |             |
 | ----------- | ----------- |
```

Comment:
as long as we're updating this, here's a new table to replace the markdown one with the emoty header row
<table>
<tr>
<td>
Welcome to Sui, a next-generation smart contract platform with high throughput, low latency, and an asset-oriented programming model powered by the [Move programming language](https://github.com/MystenLabs/awesome-move)! Find everything you need within the [Sui Developer Guides](doc/src/learn/index.md) and supporting materials below.</td>
<td>
<img src="doc/static/Sui_Icon_Brand.png" alt="sui_icon" width="200"/>
</td>
</tr>
</table>

And there is a redundant link on Explore more > docs.sui.io and then the Docs link right below it. You could remove the link on Explore